src/ui/components/Library/Team/View/TestRuns/Overview/RunResults.tsx

```diff
@@ -171,7 +171,7 @@ const FileNodeRenderer = memo(function FileNodeRenderer({
                 <TestResultListItem
                   depth={depth + 1}
                   filterByText={filterByText}
-                  key={recording.id}
+                  key={test.id + recording.id}
```

Comment:
nit: This can still result in conflicts (though unlikely) e.g. test id 123 and recording id 45 vs test id 12 and recording id 345. Given the scheme our recording ids use, I guess this is not possible.

Is there any reason not to just use test id? Not sure I understand what's going on here fully. I guess sometimes there's multiple tests per recording and other times there's multiple recordings per test?

Will there ever be many-to-many?